tests/phpunit/tests/functions/getCalendar.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * Tests for the get_calendar() function.
+ *
+ * @since 6.8.0
+ *
+ * @group functions
+ * @group calendar
+ *
+ * @covers ::get_calendar
+ */
+class Tests_Get_Calendar extends WP_UnitTestCase {
+
+	/**
+	 * Array of post IDs for testing.
+	 *
+	 * @var int[]
+	 */
+	protected static $post_ids;
+
+	/**
+	 * Set up before class.
+	 *
+	 * @param WP_UnitTest_Factory $factory Test factory.
+	 */
+	public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
+		self::$post_ids = $factory->post->create_many(
+			3,
+			array(
+				'post_date' => '2025-02-01 12:00:00',
+			)
+		);
+	}
+
+	/**
+	 * Clean up test data after class.
+	 */
+	public static function wpTearDownAfterClass() {
+		foreach ( self::$post_ids as $post_id ) {
+			wp_delete_post( $post_id, true );
+		}
+	}
```

Comment:
No need for this, `\WP_UnitTestCase_Base::tear_down_after_class()` already deletes all posts.